content/tutorial/tutorial.md

```diff
@@ -75,7 +75,7 @@ This setup requires more work but allows you to complete the tutorial using an e
 2. Follow the [installation instructions for Create React App](/docs/create-a-new-react-app.html#create-react-app) to make a new project.
 
 ```bash
-npx create-react-app my-app
+npx create-react-app@latest my-app
```

Comment:
`npx` usually pulls in the latest version of the package?

Reply:
It's causing a bug in the latest releases. Adding @latest fixes this.

Check this link for the whole issue:
https://github.com/facebook/create-react-app/issues/12022

Reply:
What exact Node and npm versions do you have and what are the exact repro instructions?